Taskfile.yml

```diff
@@ -207,3 +207,30 @@ tasks:
     cmds:
       - >-
         PYTHONPATH=$(pwd) poetry run python -W ignore utils/run_model.py {{.CLI_ARGS}}
+
+  update-requirements:
+    desc: Update the requirements.txt file for a pipeline script.
+    summary: |
+      Example usage:
+      task update-requirements -- pipeline/eval/requirements/eval.in
+    cmds:
+      # Make sure a file was given to update.
+      - >-
+        if [[ -z "{{.CLI_ARGS}}" ]]; then
+          echo "Provide a path to the .in file";
+          echo "For example:"
+          echo "task update-requirements -- pipeline/eval/requirements/eval.in";
+          exit 1
+        fi
+      # Make sure the command is being run for docker
```

Comment:
Python 3.12 breaks on my machine for pip-compile on our repo, this is why I am enforcing docker.

Reply:
Yup, that makes sense. (We do this in a bunch of other places as well for the same reason.)